ivy/functional/frontends/torch/tensor.py

```diff
@@ -173,6 +173,11 @@ def addmm_(self, mat1, mat2, *, beta=1, alpha=1):
     def addmv(self, mat, vec, *, beta=1, alpha=1):
         return torch_frontend.addmv(self, mat, vec, beta=beta, alpha=alpha)
 
+    @with_unsupported_dtypes({"2.0.1 and below": ("float16",)}, "torch")
+    def addmv_(self, mat, vec, *, beta=1, alpha=1):
+        self.ivy_array = torch_frontend.addmv(self, mat, vec, beta=beta, alpha=alpha)
+        return self
```

Comment:
We don't need to perform an inplace update on the input tensor, you can simply call the frontend function 
`return torch_frontend.addmv(*args, **kwargs)`

Reply:
@hirwa-nshuti this function is performing and inplace update based on the documentation

![image](https://github.com/unifyai/ivy/assets/18273833/a7962f01-3f16-4d4c-aa3d-86304038de12)

Should we have something similar to the documentation? 

Plus, the other functions in the file are following a similar pattern: 

https://github.com/unifyai/ivy/blob/e6d439a27573df654f7fe29e17852808ba54ba69/ivy/functional/frontends/torch/tensor.py#L162-L184

Reply:
yeah I totally agree let's just wait for CI checks

Reply:
@hirwa-nshuti I think the CI is done, should we do anything else?